src/Chapter1.hs

```diff
@@ -619,8 +631,8 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = (lastDigit (abs n)) + (lastDigit (div (abs n) 10))
```

Comment:
Great one!
Note that you can omit the parenthesis in here:

```suggestion
sumLast2 n = lastDigit (abs n) + lastDigit (div (abs n) 10)
```

Also, as you see that you use `abs n` twice in one function. That means that you could move it into a separate variable in the `let` or `where` clause and reuse everywhere 🙂

Reply:
Oh that makes sense